src/api/goods.js

```diff
@@ -0,0 +1,12 @@
+const API_URL = 'https://www.omdbapi.com/?apikey=77e39eb1&t=';
+
+export const getConnect = title => (
```

Comment:
```suggestion
export const getMovie = title => (
```
🤔 getConnect is a strange name